examples/topologies/leafnode-sys-bridge/cli/main.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/sh
+
+set -euo pipefail
+
+NATS_HUB_URL="nats://0.0.0.0:4222"
+NATS_LEAF1_URL="nats://0.0.0.0:4223"
+NATS_LEAF2_URL="nats://0.0.0.0:4224"
+
+# ### Hub setup
+# Create the operator, generate a signing key (which is a best practice),
+# and initialize the default SYS account and sys user.
+nsc add operator \
+  --generate-signing-key \
+  --sys hub
+
+# A follow-up edit of the operator enforces signing keys are used for
+# accounts as well. Setting the server URL is a convenience so that
+# it does not need to be specified with call `nsc push` while
+# the operator is set in the `nsc` environment.
+nsc edit operator \
+  --require-signing-keys \
+  --account-jwt-server-url \
+  "$NATS_HUB_URL"
+
+# For this example, we are demonstrating how we can create
+# a user which can be used in a leaf node remote and bound
+# to the leaf system account.
+nsc add account OPS
+
+nsc edit account OPS \
+  --sk generate
+
+nsc add user \
+  --account OPS ops
+
+# Finally, generate the config for the server.
+nsc generate config \
+  --nats-resolver \
+  --sys-account SYS > hub-resolver.conf
+
+# ### Leaf nodes
+# Create the operators and system accounts for the leaf nodes.
+# No additional accounts or users are required for this example.
+nsc add operator \
```

Comment:
I'm not sure if it helps this example if there is another operator.
Most users have one.